atd-vze/src/queries/notes.js

```diff
@@ -30,3 +30,30 @@ export const INSERT_NOTE = gql`
     }
   }
 `;
+
+export const UPDATE_NOTE = gql`
+  mutation UpdateNote($note: String!, $id: Int!) {
+    update_notes_by_pk(
+      pk_columns: {id: $id}
```

Comment:
I have not seen this mutation pattern where you explicitly use a primary key, but I like it! 🔑